css/style.css

```diff
@@ -0,0 +1,64 @@
+
+
+:root {
+    --blue: rgb(92, 92, 207);
+    --background: rgb(214, 211, 232);
+}
+
+
+
+* {
+    background-color: var(--background);
+}
+
+#image-container {
+    display: flex;
+    width: 100vw;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+}
+
+#image {
+    max-width: 100%; 
+    height: auto;
+}
+#main {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+h1, #img-caption {
+    width: 100vw;
+    justify-content: center;
+    text-align: center;
+    font-size: 3em;
+}
+
+#img-caption {
+    font-size: 1.5em;
+}
+
+
+
+
+
+#tribute-info {
+    justify-content: center;
+    text-align: justify;
+    padding: 0 100px;
```

Comment:
For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡